components/storage/blockdevice/COMPONENT_FLASHIAP/FlashIAPBlockDevice.h

```diff
@@ -0,0 +1,124 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2016 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MBED_FLASHIAP_BLOCK_DEVICE_H
+#define MBED_FLASHIAP_BLOCK_DEVICE_H
+
+#ifdef DEVICE_FLASH
+
+#include "BlockDevice.h"
+#include <mbed.h>
```

Comment:
this shall include what is needed instead of "all"

Reply:
Fixed